test/widgets/action_sheet_test.dart

```diff
@@ -2049,9 +2055,11 @@ void main() {
           delay: const Duration(milliseconds: 500));
         await tapCopyMessageTextButton(tester);
         // … and pump a frame to finish the NavigationState.pop animation…
-        await tester.pump(const Duration(milliseconds: 250));
+        await transitionDurationObserver.pumpPastTransition(tester);
         // … before the request finishes.  This is the repro condition for #732.
-        await tester.pump(const Duration(milliseconds: 250));
+        await transitionDurationObserver.pumpPastTransition(tester);
+        // …pump for snackbar to show
+        await tester.pumpAndSettle();
```

Comment:
How long is the snackbar entrance animation? Ideally we would pump with exactly that duration, rather than using `pumpAndSettle`.